build.gradle

```diff
@@ -34,21 +38,33 @@ repositories {
 
 dependencyManagement {
 	imports {
-		mavenBom("org.springframework.boot:spring-boot-starter-parent:2.1.8.RELEASE")
+		mavenBom("org.springframework.boot:spring-boot-starter-parent:${springBootStarterParent}")
 	}
 }
 
 dependencies {
 	compile("org.springframework.boot:spring-boot-starter-web")
 	compile('org.springframework.boot:spring-boot-starter-actuator')
 	compile('org.springframework.boot:spring-boot-starter-data-jpa')
+	compile("com.zaxxer:HikariCP:${hikariCPVersion}")
 	compile("com.h2database:h2")
 	compile('mysql:mysql-connector-java')
 	compile('org.liquibase:liquibase-core')
-	compile("org.springframework.boot:spring-boot-starter-thymeleaf")
-	compile('org.projectlombok:lombok')
-	compile("org.hibernate:hibernate-core:5.3.0.Final")
+	compile("org.springframework.boot:spring-boot-starter-thymeleaf:${thymeleafVersion}")
+	compile("org.thymeleaf:thymeleaf-spring5:${thymeleafSpringVersion}")
```

Comment:
As this is going to be "pure" REST service (we've building FE independently of server app) do we still need Thymeleaf?